golden_response/staging/get_stats/nyt_covid_cases.json

```diff
@@ -155,52 +155,56 @@
       "2020-06-20": 2268034,
       "2020-06-21": 2294413,
       "2020-06-22": 2324879,
-      "2020-06-23": 2359937,
-      "2020-06-24": 2396926,
-      "2020-06-25": 2438099,
-      "2020-06-26": 2483625,
-      "2020-06-27": 2525929,
+      "2020-06-23": 2359939,
+      "2020-06-24": 2396928,
+      "2020-06-25": 2438101,
+      "2020-06-26": 2483629,
+      "2020-06-27": 2525928,
       "2020-06-28": 2565436,
-      "2020-06-29": 2604924,
-      "2020-06-30": 2653319,
+      "2020-06-29": 2604932,
+      "2020-06-30": 2653321,
       "2020-07-01": 2703296,
-      "2020-07-02": 2758851,
-      "2020-07-03": 2816005,
-      "2020-07-04": 2866014,
-      "2020-07-05": 2910780,
-      "2020-07-06": 2958095,
-      "2020-07-07": 3012252,
-      "2020-07-08": 3071711,
-      "2020-07-09": 3131610,
-      "2020-07-10": 3199837,
-      "2020-07-11": 3260565,
-      "2020-07-12": 3318372,
-      "2020-07-13": 3379943,
-      "2020-07-14": 3445544,
-      "2020-07-15": 3513892,
-      "2020-07-16": 3589589,
-      "2020-07-17": 3660512,
-      "2020-07-18": 3722966,
-      "2020-07-19": 3785241,
-      "2020-07-20": 3845124,
-      "2020-07-21": 3910398,
-      "2020-07-22": 3980128,
-      "2020-07-23": 4050126,
-      "2020-07-24": 4123651,
-      "2020-07-25": 4190422,
-      "2020-07-26": 4244634,
-      "2020-07-27": 4303813,
-      "2020-07-28": 4366928,
-      "2020-07-29": 4433541,
-      "2020-07-30": 4502654,
-      "2020-07-31": 4570381,
-      "2020-08-01": 4628534,
-      "2020-08-02": 4677855,
-      "2020-08-03": 4725274,
-      "2020-08-04": 4778669,
-      "2020-08-05": 4832395,
-      "2020-08-06": 4889733,
-      "2020-08-07": 4950708
+      "2020-07-02": 2758855,
+      "2020-07-03": 2816009,
+      "2020-07-04": 2866015,
+      "2020-07-05": 2910782,
+      "2020-07-06": 2958098,
+      "2020-07-07": 3012182,
+      "2020-07-08": 3071637,
+      "2020-07-09": 3131526,
+      "2020-07-10": 3199753,
+      "2020-07-11": 3260474,
+      "2020-07-12": 3318279,
+      "2020-07-13": 3379846,
+      "2020-07-14": 3445448,
+      "2020-07-15": 3513790,
+      "2020-07-16": 3589477,
+      "2020-07-17": 3660400,
+      "2020-07-18": 3722851,
+      "2020-07-19": 3785126,
+      "2020-07-20": 3845014,
+      "2020-07-21": 3910291,
+      "2020-07-22": 3980030,
+      "2020-07-23": 4050036,
+      "2020-07-24": 4123562,
+      "2020-07-25": 4190338,
+      "2020-07-26": 4244555,
+      "2020-07-27": 4303736,
```

Comment:
Found this here: https://github.com/nytimes/covid-19-data/blob/e2c51040dd8d008f7fa1c6ac3ae3471dc2632c61/us.csv

The data actually changed already 😬